README.rst

```diff
@@ -25,29 +25,19 @@ Currently the following checks are available:
 Running
 -------
 
-You can run the checks locally with
-`Taskotron <https://fedoraproject.org/wiki/Taskotron>`__. First,
-install it (you can
-follow the
-`Quickstart <https://qa.fedoraproject.org/docs/libtaskotron/latest/quickstart.html>`__).
-You'll also need the ``rpm``, ``dnf`` and ``libarchive-c`` Python 2 modules
-(``python2-rpm``, ``python2-dnf``, ``python2-libarchive-c``).
-Note that Taskotron unfortunately runs on Python 2, but the code in
-this repository is Python 3 compatible as well.
-
-Once everything is installed you can run the task on a Koji build
-using the
-``name-(epoch:)version-release`` (``nevr``) identifier.
+To run this task locally, execute the following command as root (don't do this
+on a production machine!)::
 
-.. code:: console
+  $ ansible-playbook tests.yml -e taskotron_item=<nevr>
```

Comment:
Is this how you run the checks in production now instead of the `runtask` command? Will this command give the yaml output that we can parse as we did in our integration tests? We want to make sure the tests work in Taskotron not, that some ansible playbook works.

Reply:
Basically, yes. We set up more variables. Here's an example:
http://taskotron-dev.fedoraproject.org/artifacts/all/6ea80e0c-0756-11e8-920b-525400ee7c53/task_output/tests.yml/taskotron/ansible_vars.json.gz

The ResultYAML file will be stored in `{{artifacts}}/taskotron/results.yaml`, which for local execution defaults to `./artifacts/taskotron/results.yaml`. That's exactly the file we will then send to ResultsDB.

This is as close as possible to actual taskotron execution, so verifying this is useful, I believe. Of course it's not the same as running through the whole taskotron stack.

Reply:
Our integration tests now read the `results.yaml` file.